release-notes/release-notes57.md

```diff
@@ -39,6 +39,12 @@ The following enhancements were made for the {{site.productname}} 5.7 release.
 
 ### Enhancement name
 
+### Media live embeds now support video and audio elements
```

Comment:
```suggestion
### Media live embeds now supports video and audio elements
```
maybe? just read weirdly to me

Reply:
Will leave that call up to @tylerkelly13, as I prefer `support` here.

Reply:
Maybe:
> The Media Live Embeds feature now supports video and audio elements

?

Reply:
Not sure `feature` makes sense in this context...

Reply:
depends on how you read it. Media Live Embeds could be referring to the feature overall, the option, or the embedded element created using the feature + option.

Reply:
Leaving as-is